Fixes the following error caused due to binutils 2.39+.

-- snip --
arm-poky-linux-musleabi-ld.bfd: warning: .../tmp/work/cortexa32hf-neon-poky-linux-musleabi/trusted-firm
ware-a/1.0/build/devkit_e7//bl32/enable_mmu.o: missing .note.GNU-stack section implies
 executable stack
arm-poky-linux-musleabi-ld.bfd: NOTE: This behaviour is deprecated and will be removed
 in a future version of the linker
make: *** [Makefile:777:.../build/tmp/work/cortexa32hf-neon-poky-linux-musleabi/trusted-firmware-a/1.0/build
/devkit_e7//bl32/bl32.elf] Error 1
-- snip --

More info: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=ffcf9c5700e49c0aee42dcba9a12ba21338e8136

diff --git a/Makefile b/Makefile
index 3ad74759..cb46513c 100644
--- a/Makefile
+++ b/Makefile
@@ -255,7 +255,7 @@ ifneq ($(findstring armlink,$(notdir $(LD))),)
 TF_LDFLAGS		+=	--diag_error=warning --lto_level=O1
 TF_LDFLAGS		+=	--remove --info=unused,unusedsymbols
 else
-TF_LDFLAGS		+=	--fatal-warnings -O1
+TF_LDFLAGS		+=	-z noexecstack --fatal-warnings -O1
 TF_LDFLAGS		+=	--gc-sections
 endif
 TF_LDFLAGS		+=	$(TF_LDFLAGS_$(ARCH))
